Sources/Get/Fetcher.swift

```diff
@@ -82,8 +84,11 @@ extension Fetcher {
                     guard specifiedVersionRange ~= pkg.version else {
                         throw Error.UpdateRequired(url)
                     }
-                    graph[url] = (pkg, specifiedVersionRange)
-                    return try recurse(pkg.children) + [url]
+                    try rmtree(try directories().root, "Packages")
```

Comment:
An error has occured when executing `rename` without removing directories in the `Packages` directory.